src/components/FormAddClinic.js

```diff
@@ -218,22 +251,22 @@ export default function Add() {
         </div>
 
         <div className="Visitors">
-          <input type="checkbox" value={visitors} onChange={handleVisitors} />
+          <input type="checkbox" checked={visitors} onChange={handleVisitors} />
           <label for="Visitors"> Besuch erlaubt </label>
         </div>
 
         <div className="Children">
-          <input type="checkbox" value={children} onChange={handleChildren} />
+          <input type="checkbox" checked={children} onChange={handleChildren} />
           <label for="Children"> Kinder erlaubt </label>
         </div>
 
         <div className="Animals">
-          <input type="checkbox" value={animals} onChange={handleAnimals} />
+          <input type="checkbox" checked={animals} onChange={handleAnimals} />
           <label for="Animals"> Haustiere </label>
         </div>
 
         <div className="Room">
-          <input type="checkbox" value={room} onChange={handleRoom} />
+          <input type="checkbox" checked={room} onChange={handleRoom} />
           <label for="Room"> Einzelzimmer </label>
         </div>
```

Comment:
these could be components?

Reply:
Thank you! I do this in the end when everything is working :)